router/terraform/asg.tf

```diff
@@ -0,0 +1,20 @@
+module "router_cluster_asg" {
```

Comment:
I have created a separate ASG & Cluster for this service as these modules behave in an understood way with the service module - these might be shared between multiple services.